orange/src/test/java/com/hartwig/hmftools/orange/algo/purple/LossOfHeterozygositySelectorTest.java

```diff
@@ -79,7 +79,7 @@ public void canSelectGeneForLOHBasedOnGermlineDeletion() {
         String gene = LossOfHeterozygositySelector.HRD_GENES.iterator().next();
         GeneCopyNumber hrdGene = GeneCopyNumberTestFactory.builder().geneName(gene).minMinorAlleleCopyNumber(1D).minCopyNumber(2D).build();
 
```

Comment:
I would make this test independent of HRD genes for clarity of what you are testing. I assume this test will work when gene = `"gene"` as well

Reply:
Agreed, my bad